.github/workflows/deploy_dev.yml

```
@@ -0,0 +1,114 @@
+name: deploy-dev
+
+on: workflow_dispatch
+
+env:
+  # Set the IMAGE_TAG in a way that works for both pull_request and push events.
```

```suggestion
env:
  EASI_APP_NODE_VERSION: '12.22.1'
  # Set the IMAGE_TAG in a way that works for both pull_request and push events.
```
We'll need this for the `Set up node` step.